lib/Doctrine/ORM/Query/Expr.php

```diff
@@ -28,14 +28,14 @@ class Expr
      *     // (u.type = ?1) AND (u.role = ?2)
      *     $expr->andX($expr->eq('u.type', ':1'), $expr->eq('u.role', ':2'));
      *
-     * @param Expr\Comparison|Expr\Func|Expr\Orx|string $x Optional clause. Defaults to null, but requires at least one
+     * @param Expr\Comparison|Expr\Func|Expr\Orx|string ...$clauses Optional clauses. Defaults to an empty array, but requires at least one
      *                                                     defined when converting to string.
      *
      * @return Expr\Andx
      */
-    public function andX($x = null)
+    public function andX(...$clauses)
```

Comment:
This change is a BC-break for extending classes: https://3v4l.org/IkqbQ, and this class is not `final`